### PR TITLE
enable test for katib-manager

### PR DESCRIPTION
**Pull request by hougangliu**
enable test for katib-manager and fix RegisterTrial error

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/478)

### CODE REVIEW

File: manifests/v1alpha2/katib/db/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          initialDelaySeconds: 5
+          periodSeconds: 2
+          timeoutSeconds: 1
+        volumeMounts:
+        - name: katib-mysql
+          mountPath: /var/lib/mysql
+      volumes:
+      - name: katib-mysql
+        persistentVolumeClaim:
+          claimName: katib-mysql
```
Comment (gaocegege) [inline manifests/v1alpha2/katib/db/deployment.yaml]: Will we provide the PV config for users?

Comment (andreyvelich) [reply manifests/v1alpha2/katib/db/deployment.yaml]: I think, we have to provide it, like in v1alpha1. 
Some users don't have storage classes in their k8s clusters to bound PVC.

Comment (gaocegege) [reply manifests/v1alpha2/katib/db/deployment.yaml]: If some users do not have storage classes, then how can we create PV for the users? Do you mean that we use the default storage class to do it?

Comment (andreyvelich) [reply manifests/v1alpha2/katib/db/deployment.yaml]: If users don't have StorageClass they need to follow this instruction: https://github.com/kubeflow/katib#katib.
They need to create PV manually from the yaml file.

Comment (hougangliu) [reply manifests/v1alpha2/katib/db/deployment.yaml]: Ah! I missed pv/pvc.

Comment (gaocegege) [reply manifests/v1alpha2/katib/db/deployment.yaml]: Yeah, I agree. Thus we could add a default config for the PV and PVC.

Comment (hougangliu) [reply manifests/v1alpha2/katib/db/deployment.yaml]: in fact, files in manifest are used to test. In kubeflow product, only PV will not be created in ksonnet/kustimize

Comment (andreyvelich) [reply manifests/v1alpha2/katib/db/deployment.yaml]: We can create README file under kustomize Katib manifest, how to create PV, like we have here: https://github.com/kubeflow/kubeflow/tree/master/kubeflow/katib#katib-1

**Comment by hougangliu** [comment]
@YujiOshima @johnugeorge @richardsliu @gaocegege @andreyvelich 
test passed.

**Pull request merged**